platforms/core-execution/workers/src/integTest/groovy/org/gradle/workers/internal/WorkerExecutorProblemsApiIntegrationTest.groovy

```diff
@@ -120,7 +120,6 @@ class WorkerExecutorProblemsApiIntegrationTest extends AbstractIntegrationSpec {
                     );
 
                     // Write the current build operation id to a file
-                    // This needs to be Java 6 compatible, as we are in a worker
```

Comment:
Should this stay and say it needs to be compatible with Java 8?

Reply:
Two lines below the code used try-with-resources which was introduced in Java 7, so I removed the comment as it appeared stale.